android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -34,10 +34,18 @@ public MusicControlNotification(MusicControlModule module, ReactApplicationConte
         String packageName = context.getPackageName();
 
         // Optional custom icon with fallback to the play icon
-        smallIcon = r.getIdentifier("music-control-icon", "drawable", packageName);
+        smallIcon = r.getIdentifier("music_control_icon", "drawable", packageName);
```

Comment:
this change was necessary, as Android build would fail if a drawable with resource name containing a `-` was created. Using `_` resolves the issue.